modules/mobile/app/services/mobile/v0/appointments/proxy.rb

```diff
@@ -6,152 +6,17 @@ module Mobile
-          # where date ranges are not being respected
-          cc_appointments.select! do |appointment|
-            appointment.start_date_utc.between?(start_date, end_date)
-          end
-
-          facilities = fetch_facilities_from_appointments(va_appointments + va_appointment_requests)
-          va_appointments = backfill_appointments_with_facilities(va_appointments, facilities)
-          va_appointment_requests = backfill_appointments_with_facilities(va_appointment_requests, facilities)
-
-          (va_appointments + cc_appointments + va_appointment_requests + cc_appointment_requests)
-            .sort_by(&:start_date_utc)
-        end
-
-        def legacy_fetch_appointments(start_date, end_date)
-          va_response, cc_response = Parallel.map(
-            [fetch_va_appointments(start_date, end_date), fetch_cc_appointments(start_date, end_date)],
-            in_threads: 2, &:call
-          )
-
-          errors = [va_response[:error], cc_response[:error]].compact
-
-          if errors.size.positive?
-            Rails.logger.error('Mobile Legacy Appointments Error: ', errors: errors)
-            raise Common::Exceptions::BackendServiceException, 'MOBL_502_upstream_error'
-          end
-
-          va_appointments = va_appointments_adapter.parse(va_response[:response].body)
-          cc_appointments = cc_appointments_adapter.parse(cc_response[:response].body)
-
-          if va_appointments.any?
-            facilities = fetch_facilities_from_appointments(va_appointments)
-            va_appointments = backfill_appointments_with_facilities(va_appointments, facilities)
-          end
-
-          # There's currently a bug in the underlying Community Care service
-          # where date ranges are not being respected
-          cc_appointments.select! do |appointment|
-            appointment.start_date_utc.between?(start_date, end_date)
-          end
-
-          (va_appointments + cc_appointments).sort_by(&:start_date_utc)
-        end
-
-        def fetch_appointments(start_date, end_date)
-          va_response, cc_response, requests_response = Parallel.map(
-            [
-              fetch_va_appointments(start_date, end_date),
-              fetch_cc_appointments(start_date, end_date),
-              fetch_appointment_requests
-            ], in_threads: 3, &:call
-          )
-
-          # appointment requests are fetched by a service that raises on error
-          errors = [va_response[:error], cc_response[:error]].compact
-          if errors.size.positive?
-            Rails.logger.error('Mobile Appointments w/ Requests Error: ', errors: errors)
-            raise Common::Exceptions::BackendServiceException, 'MOBL_502_upstream_error'
-          end
-
-          { va: va_response, cc: cc_response, requests: requests_response }
-        end
-
-        def new_fetch_facilities(facility_ids, include_children)
+        def fetch_facilities(facility_ids, include_children)
```

Comment:
I'm not sure why we would still need this. It's a private function in a class that's going away, right?